src/main/java/Echo.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Scanner;
+
+    public class Echo {
+        private String bye;
+        private String text;
+
+    public Echo() {
+        bye ="bye";
+    }
+
+    ToDoList newList  = new ToDoList();
+
+    public void echoBack(){
+
+        Scanner in = new Scanner(System.in);
+        String text = in.next();
+
+        if(text.equals(bye)){
+            System.out.println("Bye. Hope to see you again soon!");
+        }
+        else if(text.equals(newList.checker)){
+            newList.listTask();
+            echoBack();
+        }
+        else if(text.startsWith(newList.marker)){
+            in.useDelimiter("  ");
+            String part =in.nextLine();
+            newList.mark(text+part);
+            echoBack();
+        }
+        else if(text.startsWith(newList.unmarked)){
+            in.useDelimiter("  ");
+            String part =in.nextLine();
+
+            newList.unmark(text+part);
+            echoBack();
+        }
+        else if (text.equals("todo")){
+            in.useDelimiter("  ");
+            String part =in.nextLine();
+
+            newList.addToDo(part);
+            echoBack();
+        }
+        else if (text.equals("deadline")){
```

Comment:
Perhaps the else statement could be on the same line as the closing bracket from the previous if statement?
```suggestion
        }else if (text.equals("deadline")){
```